src/main/java/com/acme/dbo/txlog/Facade.java

```diff
@@ -1,11 +1,41 @@
 package com.acme.dbo.txlog;
 
 public class Facade {
+    public static boolean isDecorated;
+
+    private static void decorateLogPrimitive(String message) {
```

Comment:
Private methods should be at the bottom of source file.

Reply:
Agreed.